README.md

```diff
@@ -142,38 +142,40 @@ Handles the style of general hi groups (see `:h highlight-args`):
-| [fern.vim](https://github.com/lambdalisue/fern.vim) | fern |
-| [dashboard-nvim](https://github.com/glepnir/dashboard-nvim) | dashboard |
-| [Markdown](https://www.markdownguide.org/) | markdown |
-| [lightspeed.nvim](https://github.com/ggandor/lightspeed.nvim) | lightspeed |
-| [leap.nvim](https://github.com/ggandor/leap.nvim) | leap |
-| [hop.nvim](https://github.com/phaazon/hop.nvim) | hop |
-| [vim-sneak](https://github.com/justinmk/vim-sneak) | vim_sneak |
-| [nvim-ts-rainbow](https://github.com/p00f/nvim-ts-rainbow) | ts_rainbow |
-| [neogit](https://github.com/TimUntersberger/neogit) | neogit |
-| [telekasten.nvim](https://github.com/renerocksai/telekasten.nvim) | telekasten |
-| [nvim-notify](https://github.com/rcarriga/nvim-notify) | notify |
-| [symbols-outline.nvim](https://github.com/simrat39/symbols-outline.nvim) | symbols_outline |
-| [mini.nvim](https://github.com/echasnovski/mini.nvim) | mini |
-| [aerial.nvim](https://github.com/stevearc/aerial.nvim) | aerial |
-| [beacon.nvim](https://github.com/DanilaMihailov/beacon.nvim) | beacon |
-| [vimwiki](https://github.com/vimwiki/vimwiki) | vimwiki |
-| [overseer.nvim](https://github.com/stevearc/overseer.nvim) | overseer |
+catppuccin-nvim provides theme support for other plugins in the Neovim ecosystem and extended Neovim functionality through _integrations_.
+
+Below is a list of supported plugins and their corresponding integration module. (See [Special integrations](https://github.com/catppuccin/nvim#special-integrations) for more.)
+
+| Plugin                                                                                | Module             |
+| ------------------------------------------------------------------------------------- | ------------------ |
+| [aerial.nvim](https://github.com/stevearc/aerial.nvim)                                | aerial             |
+| [barbar.nvim](https://github.com/romgrk/barbar.nvim)                                  | barbar             |
+| [beacon.nvim](https://github.com/DanilaMihailov/beacon.nvim)                          | beacon             |
+| [dashboard-nvim](https://github.com/glepnir/dashboard-nvim)                           | dashboard          |
+| [fern.vim](https://github.com/lambdalisue/fern.vim)                                   | fern               |
+| [gitsigns.nvim](https://github.com/lewis6991/gitsigns.nvim)                           | gitsigns           |
+| [hop.nvim](https://github.com/phaazon/hop.nvim)                                       | hop                |
+| [leap.nvim](https://github.com/ggandor/leap.nvim)                                     | leap               |
+| [lightspeed.nvim](https://github.com/ggandor/lightspeed.nvim)                         | lightspeed         |
+| [lspsaga.nvim](https://github.com/glepnir/lspsaga.nvim/)                              | lsp_saga           |
+| [Markdown](https://www.markdownguide.org/)                                            | markdown           |
```

Comment:
LGTM! Could you change "Markdown' to "markdown"

Reply:
Done!